src/material/menu/menu.scss

```diff
@@ -135,6 +135,7 @@ mat-menu {
 
   [dir='rtl'] & {
     text-align: right;
+    direction: rtl;
```

Comment:
We don't use the `direction` property, because there's no performant way to detect its usage in JS. The example in #27498 doesn't work, because the property is being set directly in the DOM which we aren't observing. Even if this gets the menu item to look correctly, other parts of the menu like the keyboard navigation might not work correctly. I'll close #27498 with an explanation.